### DISCUSSION TITLE
When should distribution type be set to download? Which benefits/drawbacks does that have?

### DISCUSSION THREAD

**Discussion by namesarebloat** [DarkMatterCore/nxdumptool#160] [Q&A] [answered]
I'm in the process of backing up my gamecards and I had to update to the rewrite, afaik this option is new and idk if I should set it or not. 

Also, what did the legacy build do?

**Comment by DarkMatterCore** [comment, marked as answer]
> I'm in the process of backing up my gamecards and I had to update to the rewrite, afaik this option is new and idk if I should set it or not.
> 
> Also, what did the legacy build do?

The distribution type is a field from NCA headers that indicates whether the NCA is supposed to be read from a gamecard (set to `1`) or not (set to `0`).

Not changing the value for the distribution type field is the default action in rewrite builds, which is the right thing to do from a preservation standpoint.

Legacy builds always modify this field -- but this didn't really affect digital dumps because their distribution type already is set to `0`.

All in all, rewrite builds give you more granular control over the dump process.
